dev/InfoBar/InfoBar.xaml

```diff
@@ -102,6 +104,7 @@
                             HorizontalAlignment="Stretch"
                             MinHeight="{ThemeResource InfoBarMinHeight}"
                             Background="{TemplateBinding Background}"
+                            contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
```

Comment:
This seems redundant as the parent (`x:Name="ContentRoot"`) of this grid already has it defined.

Reply:
You know, I thought that too at first! But if I don't set it here, setting the Background will fill in the corners.